### PR TITLE
Add support for XIAO BLE Sense boards

### PR DESCRIPTION
**Pull request by aovestdipaperino**
Added support for Xiao BLE nrf52840 boards.
IDs are taken from here: https://github.com/0hotpotman0/Adafruit_nRF52_Bootloader

Changes: 
* number of LEDs is set to 1, because a larger number causes OTA issues ¯\_(ツ)_/¯
* Set the proper SD version in the .mk files, again to fix OTA issues due to mismatched version.

### CODE REVIEW

File: src/boards/xiao_nrf52840_ble/board.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// USB
+//--------------------------------------------------------------------+
+#define USB_DESC_VID           0x2886
+#define USB_DESC_UF2_PID       0x0044
+#define USB_DESC_CDC_ONLY_PID  0x0044
+
+//------------- UF2 -------------//
+#define UF2_PRODUCT_NAME   "Seeed XIAO nRF52840"
+#define UF2_VOLUME_LABEL   "XIAO-BOOT"
+#define UF2_BOARD_ID       "Seeed_XIAO_nRF52840"
```
Comment (hathach) [inline src/boards/xiao_nrf52840_ble/board.h:65]: UF2_BOARD_ID must follow the uf2 specs format https://github.com/microsoft/uf2#files-exposed-by-bootloaders

**Review by hathach — CHANGES_REQUESTED** [review]
please correct the BOARD_ID and also add these new boards to the ci as well https://github.com/adafruit/Adafruit_nRF52_Bootloader/blob/master/.github/workflows/githubci.yml#L17

**Comment by aovestdipaperino** [comment]
Fixed all suggestions.

**Review by hathach — APPROVED** [review]
look good, thank you for your PR

**Comment by faizannazir** [comment]
only led blinking

**Pull request merged**